### ISSUE TITLE
Caltech resnet34 teacher got accuracy of 95.75%, much higher than 76.6% in the paper

### ISSUE THREAD

**Issue by MingSun-Tse** [VainF/Data-Free-Adversarial-Distillation#2]
Hi, Great thanks for your interesting work! I run your Caltech101 demo and find the teacher's accuracy is 95.75%, while in the paper table 2, it says 76.6%. I download your pretrained resnet34 teacher and prepare the data as guided in the README. Is there anything I missed? Thanks! Log is like this:
```
Teacher restored from checkpoint/teacher/caltech101-resnet34.pt
Test set: Average loss: 0.1716, Accuracy: 1623/1695 (95.7522%)
```

**Comment by VainF** [comment]
Hi @MingSun-Tse ,may be there are some problems in split_caltech.py. I will check the original code.

**Comment by VainF** [comment]
Please download the splitted dataset: https://www.dropbox.com/s/pphsk54cv8uut8d/101_ObjectCategories_split.zip?dl=0

This is my testing results:
```
Test set: Average loss: 1.0250, Accuracy: 1298/1695 (76.5782%)
```

**Comment by MingSun-Tse** [comment]
Hi @VainF , your split data works! Thank you! So what is the problem? Does the split code have some problem?

**Comment by VainF** [comment]
Maybe it is caused by the random seed.

**Comment by MingSun-Tse** [comment]
Thanks! So the code is correct, right? I plan to use it to generate my splits

**Comment by MingSun-Tse** [comment]
The code should be right. Different random seed causes overlap between the original training data and my testing data. This leads to the unusually high testing accuracy.
